src/cursor/cur_std.c

```diff
@@ -411,7 +411,7 @@ __wt_cursor_get_keyv(WT_CURSOR *cursor, uint64_t flags, va_list ap)
     } else {
         /* Fast path some common cases. */
         fmt = cursor->key_format;
-        if (LF_ISSET(WT_CURSOR_RAW_OK) || WT_STREQ(fmt, "u")) {
+        if (WT_STREQ(fmt, "u") || LF_ISSET(WT_CURSOR_RAW_OK)) {
```

Comment:
General comment: Is it worth a comment in these locations where you swapped the order to indicate someone should not change it and/or anything new should get added at the end, etc?

Reply:
Good idea, I'll add that.